generative_ai/evaluation/get_rouge_score.py

```diff
@@ -37,7 +39,37 @@ def get_rouge_score() -> EvalResult:
     life, including endangered species, it faces serious threats from
     climate change, ocean acidification, and coral bleaching."""
 
-    # Compare pre-generated model responses against the reference (ground truth).
+    # Option1: Run model inference and evaluate model response against the reference (ground truth)
```

Comment:
@msampathkumar , the reason I'm thinking about showing 2 different options of using this the computation-based metrics is that for me, as a developer, the line between these options wasn't immediately obvious (hence [this issue](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/12754) witth the "prompt" column being **silently unused**).

Reply:
While I understand your point, this code samples is still too big(100 lines). Let me check with the tech writing team.

Reply:
Also note, I don't see any example response section for this part of the code.